test/integration/model/create.test.js

```diff
@@ -1768,6 +1768,29 @@ describe(Support.getTestDialectTeaser('Model'), () => {
       });
     }
 
+    it('should support the updateOnDuplicate option', function() {
```

Comment:
only `mysql` supports `updateOnDuplicate`, make this test run if dialect is mysql https://github.com/sequelize/sequelize/blob/2b756acd8f8cddc6b63a09ee17d3a66b692841d5/lib/model.js#L2267